CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ project (gchp_ctm
   LANGUAGES Fortran CXX C
 )
 
+# Set high-level logical for GCHPctm
+set(GCHPCTM TRUE)
```

Comment:
I think project specific logicals like this should be avoided. Instead, I think GCHPctm should interact with subproject through specific variables that configure those subprojects builds to do what GCHPctm needs (i.e. top-down rather than bottom up). See my comment in the geos-chem-classic repo for more details.